schema/revert/roles/power-bi-exporter/create.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert seattleflu/id3c-customizations:roles/power-bi-exporter/create from pg
+
+begin;
+
+drop role "power-bi-exporter";
+
```

Comment:
Should there be a 'create role' in here since this file is named create.sql?

Reply:
This is a [sqitch](https://sqitch.org/) change so there are three files created: `deploy`, `revert` and `verify`. 

In this case the [deploy script](https://github.com/seattleflu/id3c-customizations/pull/117/files#diff-b168aa17ba8e63a83a7c192c28602274) creates the role, the [revert script](https://github.com/seattleflu/id3c-customizations/pull/117/files#diff-11057e04e73726cdda69d5fdfcadef1e) drops the role, and the [verify script](https://github.com/seattleflu/id3c-customizations/pull/117/files#diff-73f62be48066a6c6c8669b31ccb2760d) is empty because the database would automatically throw an error if the role couldn't be created.

Reply:
Gotcha. Thanks for explaining.